speech_recognition/pytorch/docker/Dockerfile.gpu

```diff
@@ -32,7 +32,7 @@ RUN pip install h5py \
                 hickle \
                 matplotlib \
                 tqdm \
-                http://download.pytorch.org/whl/cu80/torch-0.2.0.post3-cp27-cp27mu-manylinux1_x86_64.whl \
+                http://download.pytorch.org/whl/cu90/torch-0.4.0-cp27-cp27mu-linnux_x86_64.whl \
```

Comment:
Linux appears to be misspelled. Changing the line to `http://download.pytorch.org/whl/cu90/torch-0.4.0-cp27-cp27mu-linux_x86_64.whl` allowed me to build to image. Hope this helps.